spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -112,15 +112,21 @@ def add_group_to_build(app_id: nil, group_id: nil, build_id: nil)
     ##
 
     def post_tester(app_id: nil, tester: nil)
-      assert_required_params(__method__, binding)
+      return create_app_level_tester(app_id: app_id,
```

Comment:
I think having this api in the client makes sense, it's the same business logic that was there, I just added a new API method that allows for passing in separate args instead of just a Tester object and then I had the old API (which has a Tester object as the arg) just call into this new API. So basically, it's just a proxy to the old logic. I do think calling the client directly is bad form as you pointed out, and I should probably have a method in `Spaceship::TestFlight::Tester` proxy to the client instead.